netdata-installer.sh

```diff
@@ -790,6 +794,46 @@ cat << OPTIONSEOF
 
 OPTIONSEOF
 
+# -----------------------------------------------------------------------------
+
+copy_react_dashboard() {
+  run mv "${1}/index.html" "${1}/index-new.html"
+  run cp -r "${1}"/* "${NETDATA_WEB_DIR}"
+}
+
+install_react_dashboard() {
+  if [ -n "${NETDATA_DISABLE_REACT_DASHBOARD}" ] ; then
+    return 0
+  fi
```

Comment:
is there a way to compare latest/actual and download it only if the dashboard not installed/has newer version?

Reply:
The indexes have different checksum, so we can use it.